test/cliargsparser.cpp

```diff
@@ -250,6 +250,60 @@ TEST_CASE(
 	}
 }
 
+TEST_CASE("Sets `cache_file` when provided combined short options",
+	"[CliArgsParser]")
+{
+	const std::string filename("cache.db");
+
+	auto check = [&filename](TestHelpers::Opts opts) {
+		CliArgsParser args(opts.argc(), opts.argv());
+
+		REQUIRE(args.cache_file() == filename);
+		REQUIRE(args.lock_file() == filename + ".lock");
+		REQUIRE(args.using_nonstandard_configs());
+	};
+
+	SECTION("-vc") {
+		check({"newsboat", "-vc", filename});
+	}
```

Comment:
Since you're only checking a single case, there is no point in `SECTION` and the lambda — the test will be much shorter without them.

Reply:
I ended up with something like this, does that look about right? The test passes:
```cpp
	const std::string filename("cache.db");

	TestHelpers::Opts opts = {"newsboat", "-vc", filename};
	CliArgsParser args(opts.argc(), opts.argv());
	
	REQUIRE(args.cache_file() == filename);
	REQUIRE(args.lock_file() == filename + ".lock");
	REQUIRE(args.using_nonstandard_configs());
```

Reply:
It's still missing the check for `-v`, but other than that, it's perfect!